src/bookshelf/habitat/plan.sh

```diff
@@ -23,11 +23,19 @@ pkg_binds_optional=(
   [chef-server-ctl]="secrets"
 )
 
+# set_pwd gives a consistent current working directory when building
```

Comment:
I admit I'm a little unclear about the impact of this; why is it necessary? How does it affect local package builds (non-expeditor)

Reply:
I think tom is going to do something slightly different. Basically, with the new build, it's doing the equivalent of `build src/bifrost/habitat` instead of what we were doing before `build src/bifrost`. Hab sets different working directories for these, and our plans relied on the working directory to work correctly, so they don't work with the new build changes